visualization_scripts/environment.yml

```diff
@@ -0,0 +1,101 @@
+name: py2cyto
+channels:
+  - defaults
+dependencies:
+  - backcall=0.1.0=py35_0
```

Comment:
Do we know the minimum set of packages required to specify this environment?  Specifying specific versions of all packages is important of our testing and development.  However, sometimes the complete environment with all package versions specified can't be installed on new systems or across platform.